packages/babel-plugin-transform-regenerator/package.json

```diff
@@ -23,7 +22,19 @@
   },
   "devDependencies": {
     "@babel/core": "workspace:^",
-    "@babel/helper-plugin-test-runner": "workspace:^"
+    "@babel/helper-plugin-test-runner": "workspace:^",
+    "@babel/plugin-proposal-function-sent": "workspace:^",
+    "@babel/plugin-transform-arrow-functions": "workspace:^",
+    "@babel/plugin-transform-block-scoping": "workspace:^",
+    "@babel/plugin-transform-classes": "workspace:^",
+    "@babel/plugin-transform-for-of": "workspace:^",
+    "@babel/plugin-transform-modules-commonjs": "workspace:^",
+    "@babel/plugin-transform-parameters": "workspace:^",
+    "@babel/plugin-transform-runtime": "workspace:^",
+    "babel-check-duplicated-nodes": "^1.0.0",
```

Comment:
Is this the old `@babel/helper-check-duplicate-nodes`, let's replace it with `@babel/helper-check-duplicate-nodes`?